pkg/components/prometheus-operator/component.go

```diff
@@ -46,9 +46,11 @@ type CoreDNS struct {
 }
 
 type component struct {
-	GrafanaAdminPassword string   `hcl:"grafana_admin_password,attr"`
-	Namespace            string   `hcl:"namespace,optional"`
-	EtcdEndpoints        []string `hcl:"etcd_endpoints,optional"`
+	GrafanaAdminPassword string `hcl:"grafana_admin_password,attr"`
+	GrafanaIngressHost   string `hcl:"grafana_ingress_host,optional"`
```

Comment:
What happens when this knob isn't specified? Should this be documented in any way?

Reply:
If user does not specify anything then ingress won't be created for the user.

Reply:
Makes sense. I'll leave it up to you to decide whether we should document this fact. AFAICT this isn't self-evident.

Reply:
Added this line in the docs: `If left empty Ingress is not created.`

Reply:
Also added it in the code, template.